core/new-gui/src/app/workspace/component/workspace.component.ts

```diff
@@ -121,7 +123,7 @@ export class WorkspaceComponent implements AfterViewInit, OnDestroy {
       .pipe(debounceTime(100))
       .pipe(untilDestroyed(this))
       .subscribe(() => {
-        if (this.userService.isLogin() && this.workflowPersistService.isWorkflowPersistEnabled()) {
+        if (this.userService.isLogin() && this.workflowPersistService.isWorkflowPersistEnabled() && this.workflowCollabService.checkLockGranted()) {
```

Comment:
`checkLockGranted` usually indicates the method would fail if the check fails (like `assert`). please consider changing it to `isLockGranted`.

Reply:
I see you are using `isXXX` as variable names. you can decide the naming convension.

Reply:
Changed, now `isXXX` is only for methods.